.travis.yml

```diff
@@ -89,11 +94,18 @@ install:
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then ulimit -S -n 2048; fi
-  - echo $MAIN_CMD $SETUP_CMD
-  - eval $MAIN_CMD $SETUP_CMD
+  - export COVERAGE_FILE=$NOSE_COVERAGE1
+  - echo $MAIN_CMD $NOSE1_TEST_LIST $TEST_FLAGS $SETUP_CMD
+  - eval $MAIN_CMD $NOSE1_TEST_LIST $TEST_FLAGS $SETUP_CMD
+  - export COVERAGE_FILE=$NOSE_COVERAGE2
+  - echo $MAIN_CMD $NOSE2_TEST_LIST $TEST_FLAGS $SETUP_CMD
+  - eval $MAIN_CMD $NOSE2_TEST_LIST $TEST_FLAGS $SETUP_CMD
 
 after_success:
-  - if [[ $COVERALLS == 'true' ]]; then coveralls; fi
+  - if [[ $COVERALLS == 'true' ]]; then \
+      coverage combine $NOSE_COVERAGE1 $NOSE_COVERAGE2;
```

Comment:
there is a missing line break here. You should also use the multiline yaml marker `|`. It should be already used in the install section. So you can look there how to use it.